server/src/test/java/org/opensearch/index/query/RangeQueryBuilderTests.java

```diff
@@ -440,14 +440,23 @@ public void testDateRangeQueryTimezone() throws IOException {
             + "        }\n"
             + "    }\n"
             + "}";
+
+        // TODO what else can we assert
         QueryShardContext context = createShardContext();
         Query parsedQuery = parseQuery(query).toQuery(context);
-        assertThat(parsedQuery, instanceOf(DateRangeIncludingNowQuery.class));
-        parsedQuery = ((DateRangeIncludingNowQuery) parsedQuery).getQuery();
+
         assertThat(parsedQuery, instanceOf(ApproximateScoreQuery.class));
-        parsedQuery = ((ApproximateScoreQuery) parsedQuery).getApproximationQuery();
```

Comment:
Can you please fix the spotless (coming from https://build.ci.opensearch.org/job/gradle-check/59201/console), you can you `./gradlew spotlessApply`.